pkg/rbd/rbd_util.go

```diff
@@ -126,8 +127,27 @@ type rbdSnapshot struct {
 	RequestName    string
 }
 
+// ImageFeature represents required image features and value
+type ImageFeature struct {
```

Comment:
This probably does not need to be a public type, right?

Reply:
yes, I will fix this. Thanks!